src/rollup/RollupCreator.sol

```diff
@@ -82,30 +93,24 @@ contract RollupCreator is Ownable {
      * @param _batchPoster The address of the batch poster, not used when set to zero address
      * @param _validators  The list of validator addresses, not used when set to empty list
      * @param _nativeToken Address of the custom fee token used by rollup. If rollup is ETH-based address(0) should be provided
+     * @param _deployFactoriesToL2 Whether to deploy L2 factories using retryable tickets. If true, retryables need to be paid for in native currency.
+     *                             Deploying factories via retyrable tickets at rollup creation time is the most reliable method to do it since it
```

Comment:
```suggestion
     *                             Deploying factories via retryable tickets at rollup creation time is the most reliable method to do it since it
```